extensions/shared/components/upgrade-nudge/index.jsx

```diff
@@ -14,10 +13,9 @@ import { useEffect } from '@wordpress/element';
  */
 import analytics from '../../../../_inc/client/lib/analytics';
 import BlockNudge from '../block-nudge';
-import getSiteFragment from '../../get-site-fragment';
-import { isSimpleSite } from '../../site-type-utils';
-import './store';
+import { getUpgradeUrl } from "../../paid-blocks/utils";
 
+import './store';
```

Comment:
What changed here?

Reply:
added by mistaken, nice catch! Pretty sure it was added by the IDE 🤦